RecoLocalTracker/SiPixelRecHits/src/PixelCPEBase.cc

```diff
@@ -43,7 +43,8 @@ PixelCPEBase::PixelCPEBase(edm::ParameterSet const& conf,
       theFlag_(flag),
       magfield_(mag),
       geom_(geom),
-      ttopo_(ttopo) {
+      ttopo_(ttopo),
+      warnings_(std::string("PixelCPEBase"), edm::isDebugEnabled()) {
```

Comment:
do you actually want the debug flag to be controlled by whether `LogDebug` is turned on?

Reply:
Yes I think this behavior works fine